bluepysnap/edges.py

```diff
@@ -901,3 +901,8 @@ def iter_connections(
         else:
             omit_edge_count = lambda x: x[:2]
             return map(omit_edge_count, it)
+
+    @property
+    def h5_filepath(self):
```

Comment:
I wonder, is the h5 prefix needed here?

Reply:
I'm matching the names of what is called; uniformity is good, I think.  If we ever want to do the `CSV` stuff, (shiver), it's safer to have them named.